integration/resources/templates/combination/connector_appsync_to_eventbus.yaml

```diff
@@ -0,0 +1,173 @@
+Resources:
+  EventBus:
+    Type: AWS::Events::EventBus
+    Properties:
+      Name: !Sub "${AWS::StackName}-EventBus"
+
+  EventBridgeRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Action:
+          - sts:AssumeRole
+          Principal:
+            Service:
+            - appsync.amazonaws.com
+            - lambda.amazonaws.com
+
+  AppSyncApi:
+    Type: AWS::AppSync::GraphQLApi
+    Properties:
+      Name: AppSyncApi
+      AuthenticationType: API_KEY
+
+  ApiSchema:
+    Type: AWS::AppSync::GraphQLSchema
+    Properties:
+      ApiId: !GetAtt AppSyncApi.ApiId
+      Definition: |
+        type Query {
+          sayHello: String!
+        }
+        schema {
+          query: Query
+        } 
+
+  AppSyncEventBusDataSource:
+    Type: AWS::AppSync::DataSource
+    Properties:
+      ApiId: !GetAtt AppSyncApi.ApiId
+      Name: AppSyncEventBusDataSource
+      Type: AMAZON_EVENTBRIDGE
+      ServiceRoleArn: !GetAtt EventBridgeRole.Arn
+      EventBridgeConfig:
+        EventBusArn: !GetAtt 'EventBus.Arn'
+
+  AppSyncSayHelloResolver:
+    DependsOn: ApiSchema
+    Type: AWS::AppSync::Resolver
+    Properties:
+      ApiId: !GetAtt AppSyncApi.ApiId
+      TypeName: Query
+      FieldName: sayHello
+      DataSourceName: !GetAtt AppSyncEventBusDataSource.Name
+      RequestMappingTemplate: |
+        {
+          "version" : "2018-05-29", 
+          "operation" : "PutEvents",
+          "events" : [{}]
```

Comment:
Is it okay that `events` are empty? Can we test with that?

Reply:
Related to https://github.com/aws/serverless-application-model/pull/3311/files#r1300549275, will be resolved with verification of event being in the eventBus.